src/Adapters/Auth/apple.js

```diff
@@ -54,18 +65,44 @@ const verifyIdToken = async ({ token, id }, clientID) => {
       `auth data is invalid for this user.`
     );
   }
-  if (clientID !== undefined && jwtClaims.aud !== clientID) {
+
+  if (
+    typeof clientIDs === 'undefined' ||
+    clientIDs === undefined ||
+    clientIDs === null
+  ) {
```

Comment:
`if (!clientIDs)` should be enough. You can write a test case for both undefined and null to check.